webapp/src/shared/components/Snackbar/Snackbar.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+
+import { Text, TextVariant } from '../index';
+
+import './Snackbar.css';
+
+type SnackbarSlidePositions = 'top' | 'bottom';
+
+type SnackbarType = 'warning' | 'info';
+
+type SnackbarProps = {
+  visible: boolean;
+  text: string;
+  closeOnTap?: boolean;
+  position: SnackbarSlidePositions;
+  type: SnackbarType;
+};
+
+export default function Snackbar({
+  visible,
+  text,
+  position,
+  type,
+  closeOnTap = false,
+}: SnackbarProps) {
+  const [shouldRender, setShouldRender] = React.useState(false);
+
+  React.useEffect(() => {
+    if (visible) {
+      setShouldRender(true);
+    }
+  }, [visible]);
+
+  const onAnimationEnd = () => {
+    if (!visible) {
+      setShouldRender(false);
+    }
+  };
+
+  return (
+    <>
+      {shouldRender && (
```

Comment:
React makes kind of tricky to add animations on unMount state, because it directly removes elements of the DOM when they don't have to be rendered anymore. So we need an intermediate condition which will be set to false once the animation ends. Got some inspiration from -> https://czaplinski.io/blog/super-easy-animation-with-react-hooks/